matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java

```diff
@@ -0,0 +1,469 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.matpower.converter;
+
+import com.google.auto.service.AutoService;
+import com.google.common.io.ByteStreams;
+import com.powsybl.commons.datasource.DataSource;
+import com.powsybl.commons.datasource.ReadOnlyDataSource;
+import com.powsybl.iidm.ConversionParameters;
+import com.powsybl.iidm.import_.Importer;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.parameters.Parameter;
+import com.powsybl.iidm.parameters.ParameterDefaultValueConfig;
+import com.powsybl.iidm.parameters.ParameterType;
+import com.powsybl.matpower.model.*;
+import org.jgrapht.UndirectedGraph;
+import org.jgrapht.alg.ConnectivityInspector;
+import org.jgrapht.graph.Pseudograph;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.Month;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.*;
+
+/**
+ * @author Christian Biasuzzi <christian.biasuzzi@techrain.eu>
+ */
+@AutoService(Importer.class)
+public class MatpowerImporter implements Importer {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MatpowerImporter.class);
+
+    private static final String FORMAT = "MATPOWER";
+
+    private static final String[] EXTENSIONS = {"m", "mat"};
+
+    public static final LocalDate DEFAULTDATE = LocalDate.of(2020, Month.JANUARY, 1);
```

Comment:
The default date should be the _now()_

Coding style: a general remark about constant. The order we have is public first, then private constant.

Reply:
thanks for your feedback,
- fixed the default date: now it's now()
- aligned the constant declarations order to the coding style